vxquery-core/src/main/java/org/apache/vxquery/xmlquery/ast/ASTTag.java

```diff
@@ -80,6 +81,7 @@
     AXIS_STEP,
     NAME_TEST,
     FILTER_EXPRESSION,
```

Comment:
Do we still need the FILTER_EXPRESSION or can that we expressed by the POSTFIX_EXPRESSION as well?